cfme/tests/storage/test_volume_backup.py

```diff
@@ -68,10 +68,5 @@ def test_storage_volume_backup_edit_tag_from_detail(backup):
 @pytest.mark.tier(3)
 @pytest.mark.uncollectif(lambda: version.current_version() < '5.9')
 def test_storage_volume_backup_delete(backup):
-    """ Volume backup deletion method not support by 5.8;
-        Implementing collective delete
-    """
-
-    backups = backup.parent.all()
-    backup.parent.delete(*backups)
+    backup.parent.delete(backup)
```

Comment:
Please restore the docblock that was removed - code change is fine.

Reply:
My bad @mshriver; added info that its not supported by 5.8;